client/worker.go

```diff
@@ -52,16 +52,21 @@ func newWorker(client *http.Client, endpoint string, accessToken string) *worker
 	return w
 }
 
-func (w *worker) export(ctx context.Context, batch *jaegerpb.Batch) *ErrSend {
+func (w *worker) export(ctx context.Context, batches []*jaegerpb.Batch) *ErrSend {
 	_, span := trace.StartSpan(ctx, "export")
```

Comment:
Unrelated but good oppurtunity to assign the returne context back to `ctx` instead of ignoring.

Reply:
```golang
 	ctx, span := trace.StartSpan(ctx, "export")
```

Reply:
I see there are more places this needs to be fixed. If it is too many, I can open a PR on Monday to fix it. Feel free to ignore.

Reply:
yes. looks like it deserves another PR

Reply:
actually I'm wrong, looks like it's the only please. changed

Reply:
I think there should be a couple more in this file. At least one more in the send function.

Reply:
But the ctx not being used in those

Reply:
yea but i think we should still not ignore it so that future code uses the correct context.